plugin/build.gradle.kts

```diff
@@ -37,7 +37,7 @@ versionCompatibility {
     tests {
         dimensions {
             register("Gradle") {
-                versions = listOf("7.0", "7.1", "7.2", "7.3", "7.4", "7.5.1", "7.6", "8.0.2", "8.1.1", "8.2.1", "8.3", GradleVersion.current().version)
+                versions = listOf("7.0", "7.1", "7.2", "7.3", "7.4", "7.5.1", "7.6", "8.0.2", "8.1.1", "8.2.1", "8.3", “8.4”)
```

Comment:
The quotes are the wrong kind, https://www.compart.com/en/unicode/U+201C, and https://www.compart.com/en/unicode/U+201D respectively.